### DISCUSSION TITLE
[Plugins Tab] Sort plugins by name in Plugins Tab

### DISCUSSION THREAD

**Discussion by kookma** [TiddlyWiki/TiddlyWiki5#5256] [Ideas]
It seems in Plugins tab of $:/ControlPanel the installed plugins are sorted by publisher name, while its better to sort them by plugin name. See below screenshot

![image](https://user-images.githubusercontent.com/830394/101906498-4c0aa580-3bce-11eb-94bb-e7b7e0489ee1.png)

**Comment by Jermolene** [comment]
I agree. The rationale for sorting by title was to keep plugins from a particular publisher together in the listings, which may not be an issue in practice.

I couldn't discover how to convert this post into an issue, but I'll see what happens if I make a commit marked as fixing this post.

**Reply by kookma** [reply]
@Jermolene - Thank you
I see in drop down menu top right of the post an item as `Reference in new issue` and I just tried that. But I cannot see the backlink here! I will read the GitHub help to see how this is possible.

Ref issue: https://github.com/Jermolene/TiddlyWiki5/issues/5259

>I couldn't discover how to convert this post into an issue, but I'll see what happens if I make a commit marked as fixing this post.

Great!

**Reply by kookma** [reply]
> I couldn't discover how to convert this post into an issue, but I'll see what happens if I make a commit marked as fixing this post.

Should I create an issue for this?

**Comment by silvyn** [comment]
Great to see ref issue already close 👍 
(but indeed no real link or visual with this discussion).

**Reply by kookma** [reply]
This issue has been addressed in https://github.com/Jermolene/TiddlyWiki5/pull/5693

**Comment by linonetwo** [comment]
I think it will be better to have a search in this panel.

**Reply by kookma** [reply]
Hi @linonetwo - I think @saqimtiaz is working on some tolls lets search in Settings!
